packages/fxa-auth-server/lib/routes/devices-and-sessions.js

```diff
@@ -10,9 +10,10 @@ const ajv = new Ajv();
 const hex = require('buf').to.hex;
 const error = require('../error');
 const fs = require('fs');
-const isA = require('@hapi/joi');
+const isA = require('joi');
 const path = require('path');
 const validators = require('./validators');
+const Joi = require('joi');
```

Comment:
See comment above. This file now imports `joi` twice, using two aliases (`isA` and `Joi`).